src/SignCheck/Microsoft.SignCheck/Verification/SignatureVerificationResult.cs

```diff
@@ -188,6 +193,7 @@ public SignatureVerificationResult(string path, string parent)
 
             Filename = Path.GetFileName(path);
             FullPath = Path.GetFullPath(path);
+            VirtualPath = !string.IsNullOrEmpty(virtualPath) ? virtualPath.Replace('\\', '/') : virtualPath;
```

Comment:
Hmm, this is *technically* slightly different behavior (calls `Replace` on empty string), but same results as far as I can tell, and a little easier to understand on first glance:
```suggestion
            VirtualPath = virtualPath?.Replace('\\', '/');
```